datasets/gov/vmi/gyventoju_islaidos/vmi.py

```diff
@@ -0,0 +1,187 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.14.1
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %%
+import pandas as pd
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+import seaborn as sns
+import numpy as np
+from sklearn.decomposition import PCA
+from sklearn.preprocessing import StandardScaler
+# %matplotlib inline
+
+# %% [markdown] tags=[]
+# # IŠLAIDOS
+
+# %% [markdown]
+# Agreguoti duomenys iš juridinių asmenų pateiktos informacijos apie gyventojų patirtas įmokas ir gyventojų deklaruotas išlaidas,
+# kuriomis gali būti mažinamos gyventojų apmokestinamosios pajamos. 
+# https://data.gov.lt/dataset/gyventoju-patirtos-islaidos
+
+# %%
+islaidos=pd.read_csv("data/datasets/gov/vmi/gyventoju_islaidos/GyventojuIslaidos.csv")
+
+# %% [markdown] tags=[]
+# # KLAUSIMAI
+
+# %%
+# Kodėl daug nulinių reikšmių
+# Pataisyti brandos lygiai (visi)
+# Kodėl yra maksimali riba 600000
+
+# %% [markdown] tags=[]
+# # Duomenų apžvalga
+
+# %%
+pd.options.display.max_colwidth = 500
+islaidos.head(5)
+
+# %% [markdown]
+# Kai kurie Dtype neatitinka tipo nurodyto struktūros apraše. Pvz: sav_kodas, 
+
+# %%
+islaidos.info() 
+
+# %%
+islaidos.nunique()
+
+# %% [markdown] tags=[]
+# # Išlaidų rūšys pagal populiarumą
+
+# %% [markdown]
+# Brėžiame diagramą, kuri vaizduoja kiekvienos išlaidų rūšies dažnį šiuose duomenyse.
+
+# %%
+mpl.rc('figure', figsize=(20,10))
+islaidos_dazniai=islaidos.value_counts(['ir_kodas','pavadinimas']).sort_values(ascending = False).to_frame()
```

Comment:
```suggestion
islaidos_dazniai=islaidos['pavadinimas'].value_counts().sort_values(ascending = False).to_frame()
```

Neaišku, kokiu tikslu įtraukiamas rūšies kodas, dėjo naudojimo prastėja skaitomumas.